.gitignore

```diff
@@ -24,6 +24,10 @@ cachegrind.out*
 *.kdev4
 # Codelite (among others) project files
 *.project
+# ctags index files
+tags
+# GDB initialization scripts
+.gdbinit
```

Comment:
For debugging session I like to make a local .gdbinit, similar to:
```
file bin/pba-d-01-kw2x/usbtest.elf

dir .
set prompt (cm4-gdb) 
set print pretty

# connect to openOCD running on gdb port 3333
target remote localhost:3333

set arm force-mode thumb
```
and start openocd with ```make debug-server```. So I can use cgdb as a gdb frontend and incidentally also see logs of openocd.